Containerfile

```diff
@@ -11,6 +11,25 @@ ARG APP_ROOT=/app-root
 RUN microdnf install -y --nodocs --setopt=keepcache=0 --setopt=tsflags=nodocs \
     python3.11 python3.11-devel python3.11-pip
 
+# tar gzip are required for OpenShift CLI installation
+RUN microdnf install -y tar gzip
+
+# conditional installation of OpenShift CLI
+RUN if [ -f /cachi2/output/deps/generic/openshift-clients.tar.gz ]; then \
+      echo "Using pre-fetched OpenShift CLI from /cachi2"; \
+      tar -xvf /cachi2/output/deps/generic/openshift-clients.tar.gz -C /usr/local/bin; \
+    else \
+      echo "Pre-fetched OpenShift CLI not found. Downloading via curl."; \
+      curl -LO "https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz" && \
```

Comment:
should not we use the oc version for given OpenShift version?

Reply:
Well, how? Then we would need to install the oc at the runtime/operator deployment?